lerna.json

```diff
@@ -1,5 +1,6 @@
 {
   "npmClient": "yarn",
+  "useNx": false,
```

Comment:
This seems required for now as it seems to be breaking the releaser `check-npm` step currently.

Will track that in a different issue.